tests/testthat/test-erf_dgumbel.R

```diff
@@ -0,0 +1,8 @@
+x=rweibull(1,3)
+mu=rnorm(1)
+sigma=rweibull(1,1,3)
+test_that("erf_pnorm works", {
+  expect_true(erf_dgumbel(x=x,mu=mu,sigma=sigma)>=0)
```

Comment:
Tem uns probleminhas de formatação (não só aqui), que se vc já mandasse ajustado pouparia tempo
```suggestion
  expect_true(erf_dgumbel(x = x, mu = mu, sigma = sigma) >= 0)
```

Reply:
Irei ajustar